css/properties/max-width.json

```diff
@@ -53,52 +53,137 @@
             "deprecated": false
           }
         },
-        "sizing_values": {
+        "max-content": {
```

Comment:
Has this file been alphabetized or is this a name change?

Reply:
I added the features to the file